src/notation/view/internal/dynamicpopupmodel.h

```diff
@@ -0,0 +1,57 @@
+#ifndef MU_NOTATION_DYNAMICPOPUPMODEL_H
+#define MU_NOTATION_DYNAMICPOPUPMODEL_H
+
+#include <QList>
+#include <QObject>
+#include <QVariantList>
+
+#include "view/abstractelementpopupmodel.h"
+
+namespace mu::notation {
+class DynamicPopupModel : public AbstractElementPopupModel
+{
+    Q_OBJECT
+
+    Q_PROPERTY(QString fontFamily READ fontFamily CONSTANT)
+    Q_PROPERTY(QVariantList pages READ pages NOTIFY pagesChanged)
+
+public:
+    explicit DynamicPopupModel(QObject* parent = nullptr);
+
+    enum ItemType {
+        Dynamic,
+        Crescendo,
+        Decrescendo
+    };
+    Q_ENUM(ItemType)
+
+    struct PageItem {
+        engraving::DynamicType dynType;
+        double width;
+        double offset;
+        ItemType itemType;
+        const std::string text;
```

Comment:
It's slightly unusual to make a member field of a struct `const`, and in this case I see no reason to do it (and especially to mark this particular one `const` and the others not).

Reply:
Hmm will change it